packages/service-integration/src/service_integration/osparc_image_specs.py

```diff
@@ -26,7 +28,7 @@ def create_image_spec(
 
     - the image-spec simplifies building an image to ``docker compose build``
     """
-    labels = {**meta_cfg.to_labels_annotations()}
+    labels: dict[str, str | float | bool | None] = {**meta_cfg.to_labels_annotations()}
```

Comment:
I believe you want all "primitive" types. what about `int`?

Reply:
I simply used the what is defined in the typing for `ListOrDict` which is autogenerated